### PR TITLE
add Babel optional chaining plugin

### PR DESCRIPTION
**Pull request by RealSlimMahdi**
Hi @enuchi , thanks for developing this template.

Recently I experienced some issue trying to use Optional Chaining:

[Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) was not supported and causing Babel/App crash.

I added the Babel plugin to the code: the pkg in package.json as well as the plugin in .babelrc files.

### CODE REVIEW

**Comment by enuchi** [comment]
Thanks @RealSlimMahdi , happy to add this in. Can you also commit the package-lock.json file?

**Comment by RealSlimMahdi** [comment]
I added the lock file, sorry about that

File: src/client/.babelrc [hunk 1]
```diff
@@ -4,6 +4,7 @@
   ],
   "plugins": [
   	"@babel/plugin-proposal-object-rest-spread",
-  	"@babel/plugin-proposal-class-properties"
+  	"@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-optional-chaining"
```
Comment (enuchi) [inline src/client/.babelrc:8]: indentation is funny here, can you correct?

**Comment by enuchi** [comment]
No problem @RealSlimMahdi. Something is up with the lock file changes, though -- it should only have changes to the babel library.

I think you need to pull the lock file from master, uninstall the package and then install it again:
```
npm un @babel/plugin-proposal-optional-chaining
npm i -D @babel/plugin-proposal-optional-chaining
```

File: src/client/.babelrc (hunk 1 above)
Comment (RealSlimMahdi) [reply src/client/.babelrc:6]: weird... the formatter changed preset this time. I hope it's ok ?

**Comment by enuchi** [comment]
Yes, something still looks funny with the lock file -- going to merge into a separate branch and take a look.

**Comment by RealSlimMahdi** [comment]
Strange, I forked, clone, and install, didn't do really much more. But I have to admit I am not an expert with package lock file.

**Pull request merged**